addons/website/views/snippets/s_image_text.xml

```diff
@@ -6,7 +6,7 @@
         <div class="container">
             <div class="row align-items-center">
                 <div class="col-lg-6 pt16 pb16">
-                    <img src="/web/image/website.s_image_text_default_image" class="img img-fluid mx-auto" alt=""/>
```

Comment:
as explain, it will break all overridden of theme ! 
Each theme replace the xml id s_image_text_default_image with there own image:

```xml
    <record id="s_text_image_default_image" model="theme.ir.attachment">
        <field name="key">website.s_text_image_default_image</field>
        <field name="name">website.s_text_image_default_image</field>
        <field name="url">/theme_artists/static/src/img/snippets/s_text_image.jpg</field>
    </record>
    <record id="s_image_text_default_image" model="theme.ir.attachment">
        <field name="key">website.s_image_text_default_image</field>
        <field name="name">website.s_image_text_default_image</field>
        <field name="url">/theme_artists/static/src/img/snippets/s_image_text.jpg</field>
    </record>
```

Reply:
As I explained when we talked about it previously, since the illustrations need a specific url for their colors to be customizable, it cannot use the current xml_id customization system, and I currently don't have any idea how to make them both dynamic and customizable by the theme.

Reply:
yes, but xml id it a redirection to an url. It is the xml id website.s_image_text_default_image that you need to update and add your custom url in field url.

```python
curl -D -  https://www.odoo.com/web/image/website.s_image_text_default_image
HTTP/1.1 301 MOVED PERMANENTLY
server: nginx
date: Mon, 14 Sep 2020 08:16:46 GMT
content-type: text/html; charset=utf-8
content-length: 315
location: https://www.odoo.com/website/static/src/img/snippets_demo/s_image_text.png
```

Reply:
I understand that, but it still makes the url in the page non-customizable from the editor. That means that the user cannot change the colors, and I cannot use the main color when dropping the snippet in the page. While the url pointed to by the redirect could be a color-customized shape url, the editor would have no access to it.

Reply:
ok, now I catch your problem...  
Probably that the right tradeoff solution is to don't have default svg in website, and that theme replace with xpath the image wth xpath instead to use the url mechanism if svg is a 'requirement' for theme (e.g. paptic)

Reply:
OK. Removed illustrations from website snippets, themes can add illustrations if they need them with an xpath.